src/scikit_build_core/metadata/__init__.py

```diff
@@ -34,6 +34,8 @@
         "urls",
         "authors",
```

Comment:
`authors` also needs a special handling right?

Reply:
The goal here is to apply the action on strings, which is why there's type checking first, to produce nicer errors. (I missed the action here, adding it now) But yes, it will be type checked later.